packages/blade/src/components/Form/FormHint.tsx

```diff
@@ -79,16 +86,18 @@ export type FormHintProps = {
 
 const Icons = {
   error: ({ size }: { size: 'small' | 'medium' | 'large' }): ReactElement => (
-    <>
-      <InfoIcon color="feedback.icon.negative.intense" size={hintIconSize[size]} />
-      <BaseBox marginRight="spacing.2" />
-    </>
+    <InfoIcon
+      display={'block' as never}
```

Comment:
why never? this code is used in react native also no?

Reply:
I think it'll just be ignored on RN. 

We do the same on other places as well

https://github.com/razorpay/blade/blob/601cedc8ccf07984a2a2a88b92ca3eb8d28b1f84/packages/blade/src/components/Card/Card.tsx#L204

Reply:
Our RN setup is totally broken as of now, I do not currently have bandwidth to check and confirm if this works or not.

Reply:
lol makes sense. yolo